src/js/angular/security/controllers.js

```diff
@@ -731,8 +731,19 @@ securityCtrl.controller('ChangeUserPasswordSettingsCtrl', ['$scope', 'toastr', '
         $scope.loader = false;
 
         $scope.submit = function () {
-            $scope.updateUser();
-        };
+            if ($scope.noPassword) {
```

Comment:
The warning should be present only when you unset password for users which has admin role.